packages/wrangler/src/versions/rollback/index.ts

```diff
@@ -95,16 +94,11 @@ export async function versionsRollbackHandler(args: VersionsRollbackArgs) {
 	const rollbackTraffic = new Map([[versionId, 100]]);
 	printVersions([version], rollbackTraffic);
 
-	const confirm = await inputPrompt({
-		type: "confirm",
-		label: "Rollback",
-		question:
-			"Are you sure you want to deploy this Worker Version to 100% of traffic?",
-		defaultValue: args.yes, // defaultValue: false.    if --yes, defaultValue: true
-		acceptDefault: args.yes,
-	});
-
-	if (!confirm) {
+	const confirmed = await confirm(
+		"Are you sure you want to deploy this Worker Version to 100% of traffic?",
+		{ defaultValue: true }
+	);
+	if (!confirmed) {
```

Comment:
I noticed that in https://github.com/cloudflare/workers-sdk/pull/5950/files#diff-7b1535cea8d8067fdfcbc158145aa6ea986e82b46d3db6eb89feb254844f1175R161 we're throwing and I wondered if we wouldn't want to do smth similar to here, meaning `cli.cancel("....")`

Reply:
The cli cancel won't stop the rest of the execution which means it'll print "Aborting rollback...\nSuccess! Worker Version  ... has been deployed ..."

Because we'd be returning in createDeployment but not exiting the whole rollback execution flow

I tried that first :(

Reply:
kk 👍